perses/app/setup_relative_calculation.py

```diff
@@ -660,7 +660,8 @@ def run_setup(setup_options, serialize_systems=True, build_samplers=True):
                 if phase == 'vacuum':
                     endstates = False
                 else:
-                    endstates = True
+                    # TODO: Make this True when dispersed.utils.create_endstates is fixed
+                    endstates = False
```

Comment:
Do we want to output something to the logger so the user knows this too? We can say "Disabling creation of endstates with expanded cutoffs in order to support new alchemical factories."

Reply:
Yes, I agree.